src/utils/theme/index.js

```diff
@@ -24,7 +25,9 @@ export const COLORS = {
   porcelain: '#ecf0f1',
   silver: '#BDBDBD',
   waterspout: '#9DEDF1',
-  white: '#FFFFFF',
+  white: '#16202A', // almost black
+  white2: '#FFFFFF',
+  white10percent: 'rgba(255,255,255,0.1)',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the color named `white` is set to almost black (`#16202A`). This could lead to confusion in the future. Consider renaming this color to something more descriptive of its actual color.
```diff
-  white: '#16202A', // almost black
+  almostBlack: '#16202A',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->